### PR TITLE
[DOCS-6410] Update how to access Downloads in Support Portal/Hyland Community

### PR DESCRIPTION
**Pull request by anxumalo**
This update only addresses the wording in the Content Services 7.1 branch.
A future update will address changes in other product documentation.

### CODE REVIEW

File: content-services/latest/admin/import-transfer.md [hunk 1]
```diff
@@ -314,7 +314,7 @@ It supports sync mode transfer, so it can also be used by the replication servic
 
 The File System Transfer Receiver is delivered as a compressed zip file.
 
-1. Download the following file from the Alfresco Support Portal:
+1. Download the following file from the Support Portal:
 
     `alfresco-file-transfer-receiver-7.0.x.zip`
```
Comment (oliveralfresco2018) [inline content-services/latest/admin/import-transfer.md:319]: Should this be changed at this point? 
The file I am working with with AGS is Alfresco-Content-Services-Distribution-7.1.0.1.zip

File: content-services/latest/develop/sdk.md [hunk 2]
```diff
@@ -1902,7 +1902,7 @@ If you're using Windows, you'll need to use the `run.bat` script instead of `run
 
 The first matter to consider is to ensure that you have credentials for the Alfresco Private Repository 
 ([artifacts.alfresco.com](https://artifacts.alfresco.com/nexus/#welcome){:target="_blank"}), where the Alfresco artifacts are stored. Enterprise customers and partners can 
-request these credentials opening a ticket on the [Alfresco Support Portal](https://support.alfresco.com/){:target="_blank"}. 
+request these credentials opening a ticket on the [Support Portal](https://support.alfresco.com/){:target="_blank"}.
```
Comment (oliveralfresco2018) [inline content-services/latest/develop/sdk.md:1905]:
```suggestion
request these credentials by opening a ticket on the [Support Portal](https://support.alfresco.com/){:target="_blank"}. 
```

File: content-services/latest/develop/sdk.md [hunk 3]
```diff
@@ -1932,7 +1932,7 @@ In order to download the Docker images needed to work with Content Services Ente
 hosted at [Quay.io](https://quay.io/){:target="_blank"}. 
 
 The first matter to consider is to ensure that you have credentials for the Alfresco private Docker registry, where the Alfresco images are stored. Customers and partners can 
-request these credentials opening a ticket on the [Alfresco Support Portal](https://support.alfresco.com/){:target="_blank"}. 
+request these credentials opening a ticket on the [Support Portal](https://support.alfresco.com/){:target="_blank"}.
```
Comment (oliveralfresco2018) [inline content-services/latest/develop/sdk.md:1935]: same as above

**Review by oliveralfresco2018 — APPROVED** [review]
LGTM just some minor comments!

File: content-services/latest/admin/import-transfer.md (hunk 1 above)
Comment (anxumalo) [reply content-services/latest/admin/import-transfer.md:319]: Not for this artefact - it's still FSTR v7.0.0

**Pull request merged**